standard/standard/clause_7_normative_text.adoc

```diff
@@ -94,1003 +133,1167 @@ The Basic and Advanced Standardization Targets differ in the level of options an
+the angles represented as signed real number values.
+====
 
 ==== Requirements for Standardization Target 2: Basic-Quaternion
- 
+
 *Summary (Informative)*
 
-The Basic-Quaternion Target has a simple structure with no options. Position is specified as a point in an LTP-ENU frame and rotation is specified as a unit quaternion.
+The Basic-Quaternion Target has a simple structure with no options. Position is
+specified as a point in an LTP-ENU frame and rotation is specified as a unit
+quaternion.
 
 *Structure (Normative)*
 
 [[basic_quaternion_sdu]]
 .Basic Quaternion SDU
-image::diagram/Basic-Quaternion_SDU.PNG[align="center"]
+image::diagram/Basic-Quaternion_SDU.PNG[]
 
 *Requirements Class*
 
-|===
-<|*Requirements Class:* <|*Basic-Quaternion-SDU* 
-
-<s|ID |<<req_basic_quaternion_sdu,/req/basic_quaternion/sdu>>
-<s|Target Type|Implemntation of Logical Model
-<s|Description| Basic-Quaternion Logical Model SDU
-<s|Dependency| /req/global
-<s|Requirement B-TP-Lon-SDU|/req/tangent_plane_longitude/sdu 
-<s|Requirement B-TP-Lat-SDU|/req/tangent_plane_latitude/sdu 
-<s|Requirement B-TP-h-SDU|req/tangent_plane_h/sdu 
-<s|Requirement B-Quaternion-SDU|/req/orientation_quaternion/sdu 
-
-|===
- 
-
-
-[width="100%",cols="4,6"]
-|===
-<s|Requirement s|B-TP-Lon-SDU
-
-<s|ID |<<req_tangent_plane_longitude,/req/tangent_plane_longitude/sdu>>
-2+<|An instance of a GeoPose tangentPoint.longitude attribute SHALL be expressed as decimal degrees, represented as a signed real number.
-
-|===
-
-
-
-[width="100%",cols="4,6"]
-|===
-<s|Requirement s|B-TP-Lat-SDU
-
-<s|ID |<<req_tangent_plane_latitude,/req/tangent_plane_latitude/sdu>>
-2+<|An instance of GeoPose tangentPoint.latitude attribute SHALL be expressed as decimal degrees, represented as a signed real number.
-
-|===
-
-
-[width="100%",cols="4,6"]
-|===
-<s|Requirement s|B-TP-h-SDU
-
-<s|ID |<<req_tangent_plane_h,/req/tangent_plane_h/sdu>>
-2+<|An instance of a GeoPose tangentPoint.h attribute SHALL be expressed as a height in meters above the WGS-84 ellipsoid, represented as a signed real number. If the tangent point is above the WGS-84 ellipsoid, the value SHALL be positive. If the tangent point is below the WGS-84 ellipsoid, the value SHALL be negative. 
-
-|===
+[[req_basic_quaternion_sdu]]
+[requirements_class]
+====
+[%metadata]
+label:: /req/basic_quaternion/sdu
+subject:: Implementation of Logical Model
+inherit:: /req/global
+description:: Basic-Quaternion Logical Model SDU
+
+[requirement,name="B-TP-Lon-SDU",label="/req/tangent_plane_longitude/sdu",id="req1"]
+======
+======
+
+[requirement,name="B-TP-Lat-SDU",label="/req/tangent_plane_latitude/sdu",id="req2"]
+======
+======
+
+[requirement,name="B-TP-h-SDU",label="/req/tangent_plane_h/sdu",id="req3"]
+======
+======
+
+[requirement,name="B-Quaternion-SDU",label="/req/orientation_quaternion/sdu",id="req4"]
+======
+======
+====
+
+[requirement,id="req1"]
+====
+[%metadata]
+name:: B-TP-Lon-SDU
```

Comment:
All `name::` should be encoded as a caption of the block.

Reply:
Added `name`s as block captions.